### DISCUSSION TITLE
Make particles come out of a button by clicking it

### DISCUSSION THREAD

**Discussion by starpause** [matteobruni/tsparticles#5234] [Q&A]
I hope the title is descriptive enough. I have this pen started, got a long way following existing examples.

https://codepen.io/starpause/pen/oNJmwYd

- I can't figure out how to make the particles happen only when clicking the button. 
- I still want the particles to emit from where the click happens. 
- I want the particles under the button rather than on top of the button.
- I'd like the particles to shoot upwards rather than in every direction.

I did try digging in [docs](https://particles.js.org/docs/interfaces/tsParticles_Engine.Options_Interfaces_IOptions.IOptions.html#background) for how to get these things done but wasn't successful. TYSM for any help!

**Comment by matteobruni** [comment]
In that CodePen you are using the click mode `emitters` which creates an emitter on click, that spawns particles. It's an unrelated event to the button.

https://codepen.io/matteobruni/pen/BaOLEqW

I think this sample is better for your needs.
